devices.js

```diff
@@ -1216,6 +1216,15 @@ const devices = [
             return {'bottom_left': 1, 'bottom_right': 2, 'top_left': 3, 'top_right': 4, 'center': 5};
         },
     },
+    {
+        zigbeeModel: ['DTB190502A1'],
+        model: 'DTB190502A1',
+        vendor: 'Custom devices (DiY)',
+        description: '[CC2530 based ZigBee single Switch with Buttons https://databyte.ch)',
```

Comment:
Can you provide the complete url to the device? (So that others can also create this device).

Reply:
Hi

Sure, i have updated the url.